.gitignore

```diff
@@ -21,6 +21,9 @@
 
 /**/*.log
 
+/**/*.env
+/**/.env
```

Comment:
Should this just be `/**/.env*` ? I assume you're ignore `.env` files similar to https://www.npmjs.com/package/dotenv ? Or are you also trying to ignore things like docker-compose.env?

Reply:
Oh yeah I just added both just incase we eventually have something like docker-compose.env

Reply:
Maybe just add the one we need now as to prevent premature optimizations and confusion for future people that may want to check such a file into the repo?

Reply:
Sounds good. Made that change!